src/plugins/plot/Plot.vue

```diff
@@ -179,7 +179,7 @@ export default {
     this.stalenessSubscription = {};
     this.loadComposition();
   },
-  beforeUnmount() {
+  unmounted() {
```

Comment:
Let's revert this.

Reply:
this creates a leak if we revert for some reason